src/SFA.DAS.Campaign.Web/Views/Employer/Index.cshtml

```diff
@@ -1,8 +1,70 @@
 ﻿@{
+    Layout = "_Layout-campaign";
     ViewData["Title"] = "Employer";
+    ViewBag.Section = "employer";
 }
 
-<div style="height: 100px;"></div>
-
-<h1 class="heading-xl">Employer</h1>
+<div class="hero-banner">
+    <div class="grid-row">
+        <div class="grid-column-one-half">
+            <div class="hero-heading">
+                <h1 class="heading-xl hero-heading__heading">Turbo charge your business</h1>
+            </div>
+            <h2 class="heading-s hero-banner__sub-heading">Be part of the movement</h2>
+            <p class="hero-banner__copy">
+                Try apprenticeships for size - you'll grow your own talent, reduce staff turnover and enhance the skills your company need to succeed.
+            </p>
+            <h2 class="hero-banner__copy heading-xs">Call the National apprenticeship service</h2>
+            <p class="hero-banner__copy heading-xl">0800 015 0600</p>
```

Comment:
Lets have a chat about pulling this into a configurable field or constant - just incase it changes 😄

Reply:
Good idea